scripts/monitoring/cron-send-create-app.py

```diff
@@ -280,11 +81,22 @@ def send_zagg_data(build_ran, create_app, http_code, run_time):
         zgs.add_zabbix_keys({'openshift.master.app.create.code': http_code})
         zgs.add_zabbix_keys({'openshift.master.app.create.time': run_time})
 
+    zgs.send_metrics()
+    logger.info("Data sent to Zagg in %s seconds", str(time.time() - zgs_time))
+
+def curl(ip_addr, port, timeout=30):
+    """ Open an http connection to the url and read """
+    url = 'http://%s:%s' % (ip_addr, port)
+    logger.debug("curl(%s timeout=%ss)", url, timeout)
```

Comment:
While we *could* use this for that purpose, currently none of the apps we build in test create a route, so we currently don't test routes.